WcaOnRails/lib/middlewares/warden_user_logger.rb

```diff
@@ -0,0 +1,15 @@
+module Middlewares
+  class WardenUserLogger
+    def initialize(app, logger=Rails.logger)
+      @app = app
+      @logger = logger
+    end
+
+    def call(env)
+      warden_user = env['warden'].user
```

Comment:
So it's not clear to me from reading the tests that this actually works, since the test just mocks this as a `OpenStruct`, but I trust that this does work =). Just a thought though, would it be possible to write a higher level test that would involve less mocking? Or it be impossible to check if something was logged that way?